lib/tasks/build.js

```diff
@@ -3,13 +3,12 @@
 const path = require('path');
 const merge = require('merge-stream');
 const BowerPlugin = require('bower-webpack-plugin');
-const webpack = require('webpack-stream');
+const webpack = require('webpack');
+const webpackStream = require('webpack-stream');
 const combine = require('stream-combiner2');
-const streamify = require('gulp-streamify');
 const sass = require('gulp-sass');
 const sourcemaps = require('gulp-sourcemaps');
 const cleanCss = require('gulp-minify-css');
-const uglify = require('gulp-uglify');
```

Comment:
You can probably remove this from the package.json, not sure about streamify, double check that one